pkg/systemd/hwinfo.css

```diff
@@ -19,3 +19,27 @@ table.listing-ct thead th:last-child {
 tr.listing-ct-item td:last-child {
     text-align: left;
 }
+
+.ct-empty-slot > td:not(:first-child) {
+  color: #999;
+  text-align: left;
+}
+
+ .ct-empty-slot {
+  display: none;
+}
+
+ .show-all-slots .ct-empty-slot {
+  display: table-row;
+}
+
+ .ct-hardware-memory-empty-count {
+  font-size: 17px;
```

Comment:
@garrett: Is that kosher? Should we have fixed font sizes?

Reply:
(Oops! I checked GitHub on my day off. :wink:)

Yeah, we shouldn't have fixed font sizes.

PatternFly itself does use predefined px sizes, but it's better to use those predefined sizes as variables in LESS files instead of hardcoding it in CSS.

For straight-up CSS, I suggest using widgets that bring in other font sizes, PF classes, or scaling based on existing size. (In that order.) But we should not hardcode to px, unless it's a icon provided by a font. And even then, we should try to not do that.